packager/buildspec.yaml

```diff
@@ -54,6 +54,13 @@ foss:
         - rm plugin_*.go || true
         - GOOS=linux GOARCH=amd64 go generate --run plugin
 
+    aarch64_linux:
+      os: linux
+      arch: arm64
+      pre:
+        - rm plugin_*.go || true
+        - GOOS=linux GOARCH=amd64 go generate --run plugin
```

Comment:
```suggestion
        - GOOS=linux GOARCH=aarch64 go generate --run plugin
```
Typo?

Reply:
```suggestion
        - GOOS=linux GOARCH=arm64 go generate --run plugin
```

Reply:
The build will run in a amd64 container but cross compile for aarch64, so we need to do `go generate` with the arch of the container - see arm examples also